### PR TITLE
Update image to new script pod image

### PR DESCRIPTION
**Pull request by scme0**
For now I'm just using the artifactory image but we can publish to dockerhub and use that image once we've confirmed that it doesn't need any tweaks.

### CODE REVIEW

File: source/Octopus.Tentacle/Kubernetes/KubernetesPodContainerRegistry.cs [hunk 1]
```diff
@@ -36,7 +36,7 @@ public async Task<string> GetContainerImageForCluster()
 
             var tag = tagVersion?.ToString(2) ?? "latest";
 
-            return $"octopuslabs/k8s-workertools:{tag}";
+            return $"docker.packages.octopushq.com/octopusdeploy/k8s-agent-script-execution-image:{tag}";
```
Comment (APErebus) [inline source/Octopus.Tentacle/Kubernetes/KubernetesPodContainerRegistry.cs:39]: What testing have we done on this new image?

Comment (scme0) [reply source/Octopus.Tentacle/Kubernetes/KubernetesPodContainerRegistry.cs:39]: I've tested the health check, raw yaml and helm step so far. I was hoping we could get this in so we can test it with different setups that we (as developers) are using. Then we can fix things as they come up. I can probably spin up a quick kustomize test too.

Comment (APErebus) [reply source/Octopus.Tentacle/Kubernetes/KubernetesPodContainerRegistry.cs:39]: Yep, we need to make sure that this container is going to work for all k8s steps

As mentioned in Slack, I also don't like the name of this image

**Comment by scme0** [comment]
I'll remake this PR

**Pull request closed without merging**